modules/settings/assets/js/components/admin-top-bar/index.js

```diff
@@ -1,24 +1,22 @@
 import HelpIcon from '@elementor/icons/HelpIcon';
 import AppBar from '@elementor/ui/AppBar';
-import Grid from '@elementor/ui/Grid';
 import Link from '@elementor/ui/Link';
 import Toolbar from '@elementor/ui/Toolbar';
-import Typography from '@elementor/ui/Typography';
 import { __ } from '@wordpress/i18n';
 import { HELP_LINK } from '../../constants';
-import ElementorLogo from '../../icons/elementor-logo';
 
-const AdminTopBar = () => {
+export const AdminTopBar = () => {
```

Comment:
Why have we started doing it?
We mostly use `export default` in IO and SM, so why are we doing it differently here?
Imagine starting a new project and copying parts from different previous projects -- it can quickly become a mess.